src/utils/helpers/structure/generateReviewProgress.ts

```diff
@@ -16,10 +16,10 @@ const generateSectionReviewProgress = (section: SectionStateNEW) => {
 const generatePageReviewProgress = (page: PageNEW) => {
   const totalReviewable = page.state.filter((element) => element.isAssigned)
   const doneConform = totalReviewable.filter(
-    (element) => element.response?.reviewResponse?.decision === ReviewResponseDecision.Approve
+    (element) => element.thisReviewLatestResponse?.decision === ReviewResponseDecision.Approve
```

Comment:
Would this work for re-review as well?

Reply:
yep, should work.

deduce usual progress by thisReviewLatestResponse
deduce what is new that needs reviewing = get applicationResponses that are assigned to you, look at all of the thisReviewLatestResponse that have a decision, do an intersection (thisReviewLatestResponse.applicationResponseId === applicationResponse.id), and any applicationResponses that are left are to be reviewed